src/source/examples/get-feed-import-id-detail-v2.rst

```diff
@@ -0,0 +1,48 @@
+.. code-block:: javascript
+
+    GET /api/sellside/feed/import/123/detail
+    Accept: application/sellside.feedimportdetail.list-v2+json; application/json
+
+    200 OK
+    Content-Type: application/sellside.feedimportdetail.list-v2+json; charset=UTF-8
+
+  {
+    "ID": 123,
+    "userID": 345,
+    "URL": "http://feed.com/id/1",
+    "accepted": 1625219890,
+    "status": 1,
```

Comment:
Status should be a readable string. please change that, we should output the enum's text. use the tenum lib for that.

Reply:
can you point me to an example please?